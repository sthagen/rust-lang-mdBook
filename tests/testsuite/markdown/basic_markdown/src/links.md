# Links

- [Inline](https://example.com/inline)
- [Collapsed]
- [Emtpy reference][]
- [Specific reference][specific]
- <https://example.com/>
- <john@example.com>
- [Titled](https://example.com/title "with title")
- [Broken collapsed]
- [Broken reference][missing]
- [Markdown link](path/foo.md)
- [Markdown link anchor](path/foo.md#anchor)
- [Link anchor](#anchor)
- [With md in anchor](path.html#phantomdata)

[collapsed]: https://example.com/collapsed
[specific]: https://example.com/specific
