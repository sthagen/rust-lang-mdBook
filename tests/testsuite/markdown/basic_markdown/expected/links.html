<h1 id="links"><a class="header" href="#links">Links</a></h1>
<ul>
<li><a href="https://example.com/inline">Inline</a></li>
<li><a href="https://example.com/collapsed">Collapsed</a></li>
<li>[Emtpy reference][]</li>
<li><a href="https://example.com/specific">Specific reference</a></li>
<li><a href="https://example.com/">https://example.com/</a></li>
<li><a href="mailto:john@example.com">john@example.com</a></li>
<li><a href="https://example.com/title" title="with title">Titled</a></li>
<li>[Broken collapsed]</li>
<li>[Broken reference][missing]</li>
<li><a href="path/foo.html">Markdown link</a></li>
<li><a href="path/foo.html#anchor">Markdown link anchor</a></li>
<li><a href="#anchor">Link anchor</a></li>
<li><a href="path.html#phantomdata">With md in anchor</a></li>
</ul>